cms/djangoapps/contentstore/views/tests/test_group_configurations.py

```diff
@@ -675,6 +679,95 @@ def test_can_get_correct_usage_info_for_content_groups(self):
 
         self.assertEqual(actual, expected)
 
+    @ddt.data(ModuleStoreEnum.Type.split)
+    def test_can_get_correct_usage_info_with_orphan(self, module_store_type):
+        """
+        Test if content group json updated successfully with usage information even if there is
+        an orphan in content group.
+        """
+        with self.store.default_store(module_store_type):
```

Comment:
instead of this, you can pass the `module_store_type` to the `default_store` kwarg in [`CourseFactory`](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/modulestore/tests/factories.py#L125). Then you won't have this awkward indent